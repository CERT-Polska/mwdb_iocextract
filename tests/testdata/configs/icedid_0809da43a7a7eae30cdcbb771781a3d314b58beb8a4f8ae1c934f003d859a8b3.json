{"minor_version": 1, "binary_id": 3666847674, "domains": [{"cnc": "casadepartida.best"}, {"cnc": "gobuynewone.best"}, {"cnc": "cubabucas.xyz"}, {"cnc": "cubefresno.best"}], "type": "icedid"}