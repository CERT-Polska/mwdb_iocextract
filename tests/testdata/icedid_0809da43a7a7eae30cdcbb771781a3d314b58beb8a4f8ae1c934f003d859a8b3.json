{
    "binary_id": 3666847674,
    "domains": [
        {
            "cnc": "casadepartida.best"
        },
        {
            "cnc": "gobuynewone.best"
        },
        {
            "cnc": "cubabucas.xyz"
        },
        {
            "cnc": "cubefresno.best"
        }
    ],
    "minor_version": 1,
    "type": "icedid"
}