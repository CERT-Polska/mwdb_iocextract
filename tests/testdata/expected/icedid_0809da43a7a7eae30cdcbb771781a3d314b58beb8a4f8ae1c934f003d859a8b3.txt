NetLoc casadepartida.best/
NetLoc gobuynewone.best/
NetLoc cubabucas.xyz/
NetLoc cubefresno.best/
